hg-fast-export.py

```diff
@@ -122,6 +127,35 @@ def get_author(logmessage,committer,authors):
       return r
   return committer
 
+lfs_pointer_buffer_size = 1024*1024
+def generate_git_lfs_pointer(filename):
+  size = 0
+  # since Mercurial storage for large files are based around the sha1 we
+  # need to rehash the file to generate the proper key for the git-lfs
+  # pointer file.
```

Comment:
Since Mercurial storage for large files are based around sha1 and git lfs uses sha256 we need to rehash the file to generate the oid.